src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -36,47 +37,53 @@ protected struct Flags
             // r = isMetadataVirtual. 1 bit. (At least as true as isMetadataVirtualIgnoringInterfaceChanges.)
             //
             // s = isMetadataVirtualLocked. 1 bit.
+            //
+            // t = returnsVoid. 1 bit.
 
             private const int MethodKindOffset = 0;
-            private const int DeclarationModifiersOffset = 5;
-
             private const int MethodKindMask = 0x1F;
-            private const int DeclarationModifiersMask = 0x7FFFFF;
+            private const int MethodKindSize = 5;
+
+            private const int DeclarationModifiersOffset = MethodKindSize + MethodKindOffset;
+            private const int DeclarationModifiersSize = 0;
```

Comment:
>private const int DeclarationModifiersSize = 0; [](start = 12, length = 47)

This doesn't feel right. The fact that none of the existing tests caught this, makes me uncomfortable about the entire change. We should come up with some kind of test framework to reliably ensure that we are not sharing bits between different "regions".